src/styles/normalize.scss

```diff
@@ -0,0 +1,13 @@
+* {
+  margin: 0;
+  padding: 0;
```

Comment:
Don't reset margins and paddings in * selector for all elements. Reset only for the ones you use and move this to reset.scss file